test/external/external_test.go

```diff
@@ -224,13 +255,25 @@ var _ = Describe("External tests", func() {
 			Eventually(test.CommandSession.Out, time.Minute).Should(Say("Intermediate bundle complete at %s/testchart-intermediate.tar\n", tmpDir))
 		})
 
+		define.Then(`^the command says the intermediate bundle 2 is complete$`, func() {
+			Eventually(test.CommandSession.Out, time.Minute).Should(Say("Intermediate bundle complete at %s/testchart-intermediate-2.tar\n", tmpDir))
+		})
+
 		define.Then(`^the command says it is writing the hints file$`, func() {
 			Eventually(test.CommandSession.Out, time.Minute).Should(Say("Writing hints.yaml...\n"))
 		})
 
-		define.Then(`^remove the archive folder at testchart-intermediate.tar$`, func() {
-			err := os.Remove("testchart-intermediate.tar")
+		define.When(`^removing relok8s-save-cache$`, func() {
+			err := os.RemoveAll(saveCacheFolder())
+			Expect(err).ToNot(HaveOccurred())
+		})
```

Comment:
If this is just a cleanup step, you can add a second `func()` that runs after all of the regular steps.

http://www.bunniesandbeatings.com/goerkin/#cleanup-steps

Reply:
will do

Reply:
Actually this is not really a clean up. I will remove this to `clear relok8s-save-cache` because this is a pre-step before testing the cache. This ensures the first run is not using cached content, yet the cached content gets populated:
~~~go
		steps.When("clear relok8s-save-cache")
		steps.When(fmt.Sprintf("running relok8s chart move -y ../fixtures/testchart --image-patterns ../fixtures/testchart.images.yaml --to-intermediate-bundle %s/testchart-intermediate.tar", tmpDir))
...
		steps.Then("relok8s-save-cache contains expected layer")
~~~

Still, I did not know the cleanup trick for `goerkin`, and it might be useful elsewhere.